uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/App.java

```diff
@@ -105,6 +105,7 @@ public App(@NonNull final String packageName,
         this.updateAppInstaller = updateAppInstaller;
     }
 
+    // Use installOldApk method where available for installing old apk.
```

Comment:
I don't think we would need to add a comment on top of this method.  Please move this to inside the method